source/_docs/git.md

```diff
@@ -148,7 +148,7 @@ Now, requests to any `*.drush.in` server address should automatically accept the
 
 ### Checking Out Code using GUI Clients
 
-SourceTree and other Git GUI clients generally prompt for a Source URL using HTTP or HTTPS to the repository to check out the site code. Pantheon does not provide Git repository access over HTTP(s), and instead provides a "Git over SSH" URL. For example:
+BLAG BLAH BLEUSourceTree and other Git GUI clients generally prompt for a Source URL using HTTP or HTTPS to the repository to check out the site code. Pantheon does not provide Git repository access over HTTP(s), and instead provides a "Git over SSH" URL. For example:
```

Comment:
> BLAG BLAH BLEU

Not sure why this is there

Reply:
Sorry, I was demonstrating how docs changes are staged, and it looks like I hit commit.